pkg/validation/internal/csv.go

```diff
@@ -55,6 +55,9 @@ func parseCSVNameFormat(name string) error {
 	if violations := k8svalidation.IsDNS1123Subdomain(name); len(violations) != 0 {
 		return fmt.Errorf("%q is invalid:\n%s", name, violations)
 	}
+	if len(name) > k8svalidation.LabelValueMaxLength {
```

Comment:
If a CSV's name is supposed to be a label, why not just use [`IsDNS1123Label(name)`](https://github.com/kubernetes/apimachinery/blob/v0.21.2/pkg/util/validation/validation.go#L187)?

Reply:
It is used as label value so I think `IsValidLabelValue` is better. Fixed.